src/components/toniq-button/toniq-buttons.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {ToniqButton} from './toniq-button';
+
+export default {
+    title: 'Design System/Button',
+    component: ToniqButton,
+};
+
+export const Default = () => <ToniqButton text="Buy Now" />;
+
+export const Selection = () => <ToniqButton text="For Sale" type="selection" />;
```

Comment:
What's the intended use case for these different types of buttons? To me (right now) it just looks like one is smaller than the other and that's it:
![default-button](https://user-images.githubusercontent.com/1205860/166925359-d50fb271-13f2-4469-b4d7-69748e539c29.png)
![selection-button](https://user-images.githubusercontent.com/1205860/166925364-475d03cf-308d-4ac2-ab65-2e21b43541bf.png)

Reply:
If you look at the design there are different buttons, each have different styling e.g. hover state and uses across different pages.

Reply:
Can you provide an example of where this is used from the mocks? Also, where are you seeing hover states? I don't see anything like that in the mocks...

Reply:
In this case its selected state. These are different buttons with different padding, margin, etc. We also have buttons with image, as well as the default button has primary and secondary.

![Screen Shot 2022-05-06 at 9 08 10 AM](https://user-images.githubusercontent.com/99699764/167050585-40721187-a36f-4c78-89c5-91dbd8c7506f.png)
![Screen Shot 2022-05-06 at 9 07 11 AM](https://user-images.githubusercontent.com/99699764/167050586-f8ea4f2a-2d4f-44c8-8545-f4d206f5d826.png)
![Screen Shot 2022-05-06 at 9 11 03 AM](https://user-images.githubusercontent.com/99699764/167050803-f435c57a-ca00-413e-b50f-be43d1625b4c.png)

Reply:
Discussed over Slack, the "selected" button is like this, where it's toggled on or off.
![image](https://user-images.githubusercontent.com/1205860/167051678-659c1295-2cbc-4c2e-ba71-74a268fd8f9e.png)

`selection` type should be renamed to `toggle`

Reply:
done